index.bs

```diff
@@ -1410,6 +1433,15 @@ Its value is 50.
 maximum [=set/size=] of an [=attribution source=]'s [=attribution scopes/values=].
 Its value is 20.
 
+<dfn>Max length per aggregatable bucket</dfn> is a positive integer that controls
+the maximum [=string/length=] of an [=aggregatable bucket=]'s [=aggregatable bucket/bucket=].
```

Comment:
IIUC aggregatable bucket struct is only used in triggers, but this limit should apply to sources only.

Reply:
Updated this for source specific.